src/oop/user_defined_classes/hard/q3/MainTest.java

```diff
@@ -0,0 +1,77 @@
+package oop.user_defined_classes.hard.q3;
+
+import global.ObjectTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
+    // Java
+    private final String testClassName = "Cupboard";
+    private final String intAttributeName1 = "pots";
+    private final String booleanAttributeName1 = "anyClean";
+    private final String methodName = "washComplete";
+    private ObjectTest classInstance;
+
```

Comment:
I assume this would be for varying, but the varying options that would provide don't make much sense. By logic the number varied could only then be above 1 which means the boolean would always be true and it would never be modified. Which defeats the point of having it.